src/compile/selection/interval.ts

```diff
@@ -112,12 +112,15 @@ const interval:SelectionCompiler = {
     // Two brush marks ensure that fill colors and other aesthetic choices do
     // not interefere with the core marks, but that the brushed region can still
     // be interacted with (e.g., dragging it around).
+    const {fill, fillOpacity, ...stroke} = selCmpt.mark;
+    keys(stroke).forEach((k) => stroke[k] = {value: stroke[k]});
```

Comment:
I think this is actually breaking the type (as this line change stroke's value to be a valueRef). Maybe in the future, TS will notice and leads to compile error when we upgrade.  

Maybe we can do  this below instead

```
... (keys(stroke).reduce((vgStroke, k) => {
   vgStroke[k] = {value: stroke[k]};
   return vgStroke; 
}, {}); 
```